code/__DEFINES/~~bubber_defines/bloodsuckers.dm

```diff
@@ -0,0 +1,12 @@
+///Whether a mob is a Bloodsucker
+#define IS_BLOODSUCKER(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/bloodsucker))
+///Whether a mob is a Vassal
+#define IS_VASSAL(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/vassal))
+///Whether a mob is a Favorite Vassal
+#define IS_FAVORITE_VASSAL(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/vassal/vassal_types/favorite))
+///Whether a mob is a Revenge Vassal
+#define IS_REVENGE_VASSAL(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/vassal/vassal_types/revenge))
+///Whether a mob is a Monster Hunter-NOT NEEDED RIGHT NOW
+//#define IS_MONSTERHUNTER(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/monsterhunter))
+//Used in bloodsucker_life.dm
```

Comment:
Just delete this if it's not needed and nothing is using it.

Reply:
It's there because Fulpstation has specific monster hunt / bloodsucker interactions. Whilst we may not be porting monster hunter, someone else might in the future and in such an instance these would need to be uncommented.